src/o365/spfx/commands/project/project-externalize/rules/DynamicRule.ts

```diff
@@ -25,8 +24,6 @@ export class DynamicRule extends BasicDependencyRule {
           resolve(res
             .filter(x => x !== undefined)
             .map(x => x as ExternalizeEntry));
-        }, (err: any): void => {
-          reject(err);
```

Comment:
Turned that this is unreachable code since the promises always resolve, no matter what.